Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:latest
+
+ENV TZ=Europe/Kiev
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install -y \
+    cmake \
+    g++ \
+    build-essential \
+    gperf \
+    openssl \
+    libssl-dev \
+    zlib1g-dev \
+    git \
+    && rm -rf /var/lib/apt/lists/*
+
+
+RUN git clone --recursive https://github.com/tdlib/telegram-bot-api.git && \
+    cd telegram-bot-api/ && \
```

Comment:
`git clone` shouldn't happen during docker image build for several reasons which include:

- I am unable to rebuild the image without completely disabling the cache because Docker doesn't know that the repository could have received updates, so this step will be cached
- This way I am not able to build my own version of Telegram Bot API if I've modified sources

Instead, you should consider that your file is in the root directory of the project and use the context (i.e. run something like `COPY . /telegram-bot-api` inside of the `Dockerfile` and have all the waste `.dockerignore`d)

Reply:
so im thinking of adding `COPY . /telegram-bot-api`. it should be fixed.. already building the the container after i changed a few things in Dockerfile. i moved `TZ` as an ENV (not hardcoded)